QuizApp/QuizApp/DS/Quiz/QuizDataSource.swift

```diff
@@ -2,6 +2,8 @@ protocol QuizDataSourceProtocol {
 
     func getQuizes(for category: QuizCategoryDataModel) async throws -> [QuizResponseDataModel]
 
+    func getQuizes() async throws -> [QuizResponseDataModel]
```

Comment:
Functions that return a value and have no arguments should be computed properties.